src/redux/reducers/notifications.js

```diff
@@ -0,0 +1,64 @@
+import { v4 as uuidv4 } from "uuid";
+
+const initialState = {
+
+  1: {
+    id: 1,
+    active: true,
+    todo: {
+      id: 1,
+      name: "code all day long",
+      todoListId: 1
+    }
+  },
+  2:  {
+    id: 2,
+    active: false,
+    todo: {
+      id: 2,
+      name: "pakaLolo all day long",
+      todoListId: 2
+    }
+  },
+  3:  {
+    id: 3,
+    active: true,
+    todo: {
+      id: 3,
+      name: "plongée all day long",
+      todoListId: 3
+    }
+  }
+
+}
+
+const notifications = (state = initialState, action) => {
+  let id, todoListId;
+  let todos, todo;
```

Comment:
You don't need that for now

Reply:
mais si je l'enlève il me dit "id not defined" :'( :'( :'( :'( :'( :'( :'( :'(